cmd/reading/list/list.go

```diff
@@ -57,23 +69,9 @@ func NewCommand() *cobra.Command {
 			} else {
 				url = config.Conf.Clients["CoreData"].Url() + clients.ApiReadingRoute
 			}
-
-			resp, err := http.Get(url)
-			if err != nil {
-				// handle error
-				fmt.Println("An error occurred. Is EdgeX running?")
-				fmt.Println(err)
-			}
-			defer resp.Body.Close()
-
-			data, _ := ioutil.ReadAll(resp.Body)
-
 			var readings []models.Reading
-			err = json.Unmarshal(data, &readings)
+			err = utils.ListHelper(url, readings)
 			if err != nil {
-				if string(data) == "Error, exceeded the max limit as defined in config" {
-					fmt.Println("The number of readings to be returned exceeds the MaxResultCount limit defined in configuration.toml")
-				}
 				fmt.Println(err)
```

Comment:
The error is printed three times: first in `utils.ListHelper`, then here and additional time from the cobra library itself. (because RunE: is using) Lets just cobra library print the error for us.

Reply:
agree!